README.md

```diff
@@ -37,7 +37,7 @@ fastify.register(require('fastify-jwt'), {
   secret: 'supersecret'
 })
 
-fastify.addHook("onRequest", async (request, reply) => {
+fastify.addHook("preHandler", async (request, reply) => {
```

Comment:
This should be working with `onRequest`. Can you confirm?

Reply:
@mcollina > I made a quick test and this is working as it should.

Reply:
Ok, So why in my example I had to change `onRequest" to "preHandler" to make it work ? You can check gitter.

Reply:
@gfelot > I explained it here https://github.com/fastify/fastify-jwt/pull/46#discussion_r265197374